docs/Developers-Guide.md

```diff
@@ -1,9 +1,21 @@
 # Developers Guide
 
+## Setup OTP
+_A Quick guide to setting up the OpenTripPlanner project._
+
+You need Git, Maven and Java(JDK) and an IDE installed on your computer.
+
+1. Clone OpenTripPlanner from GitHub.
+2. Checkout the desired branch `git checkout dev-2.x` 
+3. Run `maven package`- this will download all dependencies, build the project and run tests.
+4. Open the project in your IDE.
+5. Import the _intellij-code-style.xml_ (IntelliJ IDE).
+
+
 ## Working on OTP in an IDE
 
 Most people writing or modifying OTP code use an Integrated Development Environment (IDE). Some of the most popular IDEs for
-Java development are [Eclipse](http://eclipse.org), [IntelliJ IDEA](https://www.jetbrains.com/idea/), and
+Java development are [IntelliJ IDEA](https://www.jetbrains.com/idea/), [Eclipse](http://eclipse.org), and
```

Comment:
We should probably mention that most core developers use IntelliJ, so people are likely to get more help from the community on that IDE.

Reply:
It is mentioned in the next paragraph, but I changed the text slightly. Personally I think this section is too long, it might give the impression that OTP development is simpler than it is. 

Should we increase the Java maximum memory parameter from `-Xmx2G` to `-Xmx4G`?

Reply:
I moved the code-style up a directory, the everything is resolved I believe.